aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -60,18 +60,36 @@ See [Sub-application configuration](xref:host-and-deploy/iis/index#sub-applicati
 | `rapidFailsPerMinute` | <p>Optional integer attribute.</p><p>Specifies the number of times the process specified in **processPath** is allowed to crash per minute. If this limit is exceeded, the module stops launching the process for the remainder of the minute.</p> | `10` |
-| `requestTimeout` | <p>Optional timespan attribute.</p><p>Specifies the duration for which the ASP.NET Core Module waits for a response from the process listening on %ASPNETCORE_PORT%.</p><p>The `requestTimeout` must be specified in whole minutes only, otherwise it defaults to 2 minutes.</p> | `00:02:00` |
+| `requestTimeout` | <p>Optional timespan attribute.</p><p>Specifies the duration for which the ASP.NET Core Module waits for a response from the process listening on %ASPNETCORE_PORT%.</p><p>In versions of the ASP.NET Core Module that shipped with the release of ASP.NET Core 2.0 or earlier, the `requestTimeout` must be specified in whole minutes only, otherwise it defaults to 2 minutes.</p> | `00:02:00` |
 | `shutdownTimeLimit` | <p>Optional integer attribute.</p><p>Duration in seconds that the module waits for the executable to gracefully shutdown when the *app_offline.htm* file is detected.</p> | `10` |
 | `startupTimeLimit` | <p>Optional integer attribute.</p><p>Duration in seconds that the module waits for the executable to start a process listening on the port. If this time limit is exceeded, the module kills the process. The module attempts to relaunch the process when it receives a new request and continues to attempt to restart the process on subsequent incoming requests unless the app fails to start **rapidFailsPerMinute** number of times in the last rolling minute.</p> | `120` |
 | `stdoutLogEnabled` | <p>Optional Boolean attribute.</p><p>If true, **stdout** and **stderr** for the process specified in **processPath** are redirected to the file specified in **stdoutLogFile**.</p> | `false` |
 | `stdoutLogFile` | <p>Optional string attribute.</p><p>Specifies the relative or absolute file path for which **stdout** and **stderr** from the process specified in **processPath** are logged. Relative paths are relative to the root of the site. Any path starting with `.` are relative to the site root and all other paths are treated as absolute paths. Any folders provided in the path must exist in order for the module to create the log file. Using underscore delimiters, a timestamp, process ID, and file extension (*.log*) are added to the last segment of the **stdoutLogFile** path. If `.\logs\stdout` is supplied as a value, an example stdout log is saved as *stdout_20180205194132_1934.log* in the *logs* folder when saved on 2/5/2018 at 19:41:32 with a process ID of 1934.</p> | `aspnetcore-stdout` |
+::: moniker-end
+::: moniker range=">= aspnetcore-2.1"
+[!INCLUDE[](~/includes/2.1.md)]
```

Comment:
Any way to minimize the repetition here?

Reply:
How do you mean ... the `moniker ranges` ... no can do unfortunately. They don't work inside table cells AFAIK.

Reply:
Yeah that's what I meant.

Reply:
```Markdown
::: moniker range="moniker-range-expression"

Versioned Content

::: moniker-end
```

A versioned zones starts with `::: moniker` on a new line and specifies a `range` property, the value of which is a valid moniker range. The zone ends with `::: moniker-end`, on its own line. All content between these two markers applies to the versions matched by the `range` expression.

### Limitations of Zones

1. You **can** use includes inside of a zone, along with any other standard Markdown or Docs-specific Markdown extensions.
1. You **can** use a zone inside of other constructs, like a blockquote or note.
1. You **cannot** use a `moniker` zone inside of another `moniker` zone. Nesting is not permitted.

Reply:
... so they don't really say explicitly if we can inject them into a table. Has it been tested ... I can test it on this PR if you want.

```
| Col1 | Col2 |
| ---- | ---- |
| Val1 | Val2 |
::: moniker range="<= aspnetcore-1.1"
| Val1 | Val2 |
::: moniker-end
::: moniker range=">= aspnetcore-2.0"
| Val1 | Val2 |
::: moniker-end
| Val1 | Val2 |
```

Reply:
@guardrex  try that. Currently the entire table is missing when I select 2.0